app/views/alert_mailer/alert.text.haml

```diff
@@ -33,7 +33,7 @@
 \
 \---
 \
-- unless @jobseeker_subscription
+- unless !JobseekerAccountsFeature.enabled? || @jobseeker_subscription
```

Comment:
```
- if JobseekerAccountsFeature.enabled? && !@jobseeker_subscription
```

Would read a whole lot better that way, but happy for you to merge for now 👍

Reply:
I thought that as well, it was just some preemptive time saving for the person who removes the feature flag eventually. I'll change it